build-system/config.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
     // Files in this testdata dir are machine-generated and are not part
     // of the AMP runtime, so shouldn't be checked.
     '!extensions/amp-a4a/*/test/testdata/*.js',
-    '!examples/*.js',
+    '!examples/**/*',
     '!examples/visual-tests/**/*',
```

Comment:
This line can go away since the one above encompasses it, correct?

Reply:
True, will remove in a follow-up.